middleware/proxy/upstream.go

```diff
@@ -182,15 +185,23 @@ func parseBlock(c *parse.Dispenser, u *staticUpstream) error {
 			}
 			u.HealthCheck.Interval = dur
 		}
+	case "header_upstream":
+		fallthrough
 	case "proxy_header":
```

Comment:
I'll phase out proxy_header for the 0.9 release since that for sure will break some people.